pages/about.vue

```diff
@@ -132,6 +132,10 @@ export default {
     color: $gray-2;
     margin-bottom: 12px;
   }
+  & &-Browser-List {
```

Comment:
[nits] これでよさそう？な気がしました><
```suggestion
  &-Browser-List {
```

Reply:
じつはこちら、最初に着手したときにそのようにしておりました。

その際、 ‘.v-application ul‘ セレクタで定義されたプロパティの方が強くなり、
左に余白が生じてしまった関係で、現在のコミット内容となっています。
※その旨のコメント記載すべきでしたね

Reply:
そうなんですね！意図があるのであればいいと思います。
ご確認ありがとうございます><

コメントあると、より丁寧でよいですね！

Reply:
こちらコメントも記載してコミットしました！

Reply:
ありがとうございます！！